ctx.go

```diff
@@ -107,6 +107,25 @@ func (c *DefaultCtx) RequestCtx() *fasthttp.RequestCtx {
 	return c.fasthttp
 }
 
+const userContextKey = "__fiber_context"
```

Comment:
Do it the way we did it back then.

https://github.com/gofiber/fiber/pull/3382/files#diff-c4dd82978a1a49bcd620aff72208bbd8114f09341d701e188c247ce3e3511117R47-L47